app/utilities/schema.py

```diff
@@ -196,18 +221,22 @@ def _load_schema_file(schema_name: str, language_code: str) -> Any:
 
 @lru_cache(maxsize=None)
 def load_schema_from_url(
-    schema_url: str, language_code: str | None
+    *, schema_url: str, language_code: str | None, parameters: ImmutableDict[str, str]
```

Comment:
Instead of passing a dict, pass kwargs?.

Reply:
The reason I didn't was https://github.com/ONSdigital/eq-questionnaire-runner/pull/1246#discussion_r1393979158 but if we get rid of the language param for schema url then agree with this 👍

Reply:
Dropped this now since only the cir variant has params